styles.css

```diff
@@ -0,0 +1,78 @@
+body { max-width: 375px; }
+
+/* Headline text */
+
+.primary-text {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  color: #ff6b00;
+  letter-spacing: 0.37px;
+}
+
+.supporting-text {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+/* Navbar */
+
+.navbar {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  background: #3c3a39;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  padding: 24px;
+}
+
+.navbar a {
+  color: #fff;
+  text-decoration: none;
+}
+
+/* Headline */
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  background: #1c1a19;
+  background-image: url(./Illustration-header.png);
```

Comment:
- [ ] :wave: I noticed you exported the image with the whole background i suggest you export it separately. reason to if we use wider screen the image can stretch
- [ ] also the image you are using belongs to the desktop template kindly take a look at the figma design

Your design
![Screenshot from 2022-06-28 16-48-31](https://user-images.githubusercontent.com/66230306/176542116-a3cfb663-57a9-423b-a127-2c17a69cd609.png)

Figma design
![Screenshot from 2022-06-28 16-45-42](https://user-images.githubusercontent.com/66230306/176541558-8da8aea0-4de8-4b87-974f-6da58cc87160.png)